class.jetpack.php

```diff
@@ -1249,6 +1264,31 @@ public static function is_version_controlled() {
 		}
 		return $is_version_controlled;
 	}
+
+	/**
+	 * Retrieves a list of allowed file types.
+	 * @return array $allowed_file_types
+	 **/
+	public static function allowed_file_types() {
+		$allowed_file_types= array();
+		$mime_types = get_allowed_mime_types();
```

Comment:
`get_allowed_mime_types()` is very much a user dependent function. So some users might have more privileges then others. See https://developer.wordpress.org/reference/functions/get_allowed_mime_types/ I think it might be safer to just assume that we never allow any html files from uploaded as well. 

Since what might happen is that the sync could be triggered by a user that has 'unfiltered_html' privileges but then the user that would want to use upload the files does not.
Although I am hoping that in that case the users privileges would be fail anyways because the Jetpack Site's permission would stop that. Still something to consider.